profiles/thurloe-configs/terraform/thurloe/config.tf

```diff
@@ -0,0 +1,42 @@
+provider "vault" {}
+
+resource "vault_token" "render_token" {
+  policies = ["${var.config_render_vault_policy}"]
+
+  renewable = true
+  ttl = "20m"
+}
+
+resource "null_resource" "config" {
+  triggers = {
+    always = "${uuid()}"
+  }
+  provisioner "local-exec" {
+    command = "ruby /workbench/render_configs.rb"
+    environment = {
+      INSTANCES = "${jsonencode(data.google_compute_instance_group.service-instances.instances)}"
+      CONFIG_BUCKET = "${var.config_bucket_name}"
+      VAULT_TOKEN = "${vault_token.render_token.client_token}"
```

Comment:
I believe that this token will get saved in the state file, which is why a low-scope, time-limited token  (and not the user's token) is appropriate.